src/content/deployment/android.md

```diff
@@ -323,11 +323,11 @@ to verify that the values are correct.
 `applicationId`
 : Specify the final, unique [application ID][].
   
-`minSdkVersion`
+`minSdk`
 : Specify the [minimum API level][] on which you designed the app to run.
   Defaults to `flutter.minSdkVersion`.
```

Comment:
@bartekpacia Should we also do a similar change in `flutter_tools`, to keep it consistent? We could make a backwards compatible change, although I'm not sure if it is worth to even rename in the tool.

Reply:
I don't have a strong opinion on it tbh. Seems very minor.